pkg/codegen/java/build.gradle.template

```diff
@@ -38,7 +38,7 @@ java {
 
 compileJava {
     options.fork = true
-    options.forkOptions.jvmArgs.addAll(["-Xmx4g"])
+    options.forkOptions.jvmArgs.addAll(["-Xmx16g"])
```

Comment:
Tested on GHA CI? I wonder if for smaller providers this causes OOM somewhere. Could use an env var. But can also merge and then edit later as appropriate.